Tests/ApolloCodegenTests/CodeGenIR/IRRootFieldBuilderTests.swift

```diff
@@ -4384,9 +4396,11 @@ class IRRootFieldBuilderTests: XCTestCase {
     expect(Array(self.computedReferencedFragments)).to(equal(expected))
   }
 
-  // MARK: - Deferred Fragments - hasDeferredFragments property
+  // MARK: - Deferred Fragments - containsDeferredFragment property
+
+  #warning("Need tests for deferredFragments property")
```

Comment:
This will be removed in the next PR (https://github.com/apollographql/apollo-ios-dev/pull/219) which takes care of the deferred fragment metadata definitions.